### ISSUE TITLE
Failed to build BR RC2 in online compiler

### ISSUE THREAD

**Issue by bentcooke** [ARMmbed/nanostack-border-router#119]
The nanostack BR RC2 tag fails to build in the online compiler with the following error:

`Error: Attempt to override undefined parameter 'nsapi.default-stack' in 'application[*]'`

**Comment by MarceloSalazar** [comment]
Can you confirm you're using the Partner Beta IDE and switched to the 5.9 OOB branch?

**Comment by bentcooke** [comment]
Yes, the Partner Beta IDE and RC2 tag is being used when I encounter this build error.

**Comment by ciarmcom** [comment]
ARM Internal Ref: IOTTHD-2608

**Comment by SeppoTakalo** [comment]
In last OOB, I verified that partner beta is only able to build against released Mbed OS versions.
How did you verify that its building 5.9 RC2 of Mbed OS?

Can you right click the mbed-os and show screen shot of versions field?

**Comment by artokin** [comment]
I can get this same kind of build error on my personal workarea when mbed-os is not updated to the mbed-os-5.9-rcX. Once Mbed OS is updated to the mbed-os-5.9-rcX the build error disappears.

Could it be that Mbed OS is not properly updated to mbed-os-5.9-rcX in online compiler?

**Comment by ashok-rao** [comment]
@bentcooke .. can you please check as per @artokin 's comments above? Thanks.

**Comment by artokin** [comment]
@SeppoTakalo has created new issue to tools: https://jira.arm.com/browse/IOTTOO-3392. Probably this issue should be duplicated to that.
